README.md

```diff
@@ -10,7 +10,24 @@ The engine that allows editors to build on top of Jest.
 
 This is only useful if you are interested in building an editor integration for Jest.
 
-For now as an end user, we'd recommend looking at either [vscode-jest](https://github.com/jest-community/vscode-jest/) or [majestic](https://github.com/Raathigesh/majestic/).
+For now as an end user, we'd recommend looking at either [vscode-jest](https://github.com/jest-community/vscode-jest/), [vscode-jest-runner](https://github.com/firsttris/vscode-jest-runner) or [majestic](https://github.com/Raathigesh/majestic/).
```

Comment:
Not sure why `majestic` is here, it is not using this package isn't it? Actually I think I am fine to get rid of this whole sentence altogether.

Reply:
agree